settings.gradle.kts

```diff
@@ -16,7 +16,8 @@ include("deprecatedPluginsTest",
     "memory-test",
     "errorprone",
     "junitJupiterParallelTest",
-    "osgi-test")
+    "osgi-test",
+    "bom")
```

Comment:
I would have expected some files to be added in `subprojects/bom/`, but I don't see these in this PR. Are they missing or is this working automagically?

Reply:
My bad, had to reset my previous commit because of a temporary change in some file and forgot to add the new file again. TGIF...

Reply:
This works automagically, you only need 

> The BOM in [#2321 (comment)](https://github.com/mockito/mockito/issues/2321#issuecomment-859573476) lists all versions with `SNAPSHOT`. Instead, I think we should list the explicit versions. We currently do that with Shipkit on tags: https://github.com/shipkit/shipkit-auto-version#usage-based-on-annotated-tag
> 
> Can we make sure that the explicit versions work with our GitHub actions integration? I am not actually sure how that would look like, but I think you should be able to run `./gradlew publishToMavenLocal -Pversion=3.12.0` and see what the result is.

Yes, this is exactly what happens. The other modules are versioned by the same mechanism, so the BOM retrieves their fixed version set by shipkit through Gradle. Just verified again that this works locally. Try it yourself!

Reply:
Sorry, not my day, this was supposed to go as reply to your other comment, but the window was still open. Really TGIF